iscservice/demo/README.rst

```diff
@@ -0,0 +1,96 @@
+Preparing computations library
+==============================
+
+Computations from an external library cannot be linked directly with
+a Motr instance. The library is supposed to have an entry function named
+``void motr_lib_init(void)``. All the computations in the library must
+have the following signature::
+
+  int comp(struct m0_buf *args, struct m0_buf *out,
+           struct m0_isc_comp_private *comp_data, int *rc)
+
+See demo/libdemo.c for examples.
+
+Loading the library
+===================
+
+With ``spiel`` command (see spiel/spiel.h and demo/util.h) the library
+can be loaded with any running Motr instance. A helper function
+``m0util_isc_api_register`` takes the library path which is (IMPORTANT!)
+expected to be the same across all the nodes running Motr.
+``m0iscreg`` utility takes the path as an input and loads the library
+into all the remote Motr instances.
+
+On successful loading of the library, the output will look like this::
+
+  $ ./m0iscreg $PWD/libdemo.so
+  m0iscreg success
+
+Demo computations
+=================
+
+Currently, we demonstrate three simple computations: ``ping``, ``min`` and
+``max``. ``m0iscdemo`` utility can be used to invoke the computations and
+see the result::
+
+  $ ./m0iscdemo
+
+  Usage: m0iscdemo [-v[v]] COMP OBJ LEN
+
+    Supported COMPutations: ping, min, max.
+
+    OBJ is two uint64 numbers in format: hi:lo.
+    LEN is the length of object (in KiB).
+
+To build the utility and ``libdemo.so`` library, run::
+
+  make isc-all
+
+Following are the steps to run the demo.
+
+Prepare rc-files
+----------------
+
+In order to start the demo programs, the rc-startup files must be created
+for each of them: ``~/.m0util/m0iscdemorc/$(hostname)`` and
+``~/.m0util/m0iscreg/$(hostname)`` with the configuration parameters
+needed to connect to Motr, for example::
+
+  $ cat ~/.m0util/m0iscdemorc/centos79
+  HA_ENDPOINT_ADDR = 192.168.180.171@tcp:12345:1:1
+  PROFILE_FID   = 0x7000000000000001:0x50
+  M0_POOL_TIER1 = 0x6f00000000000001:0x2f
+  LOCAL_ENDPOINT_ADDR0 = 192.168.180.171@tcp:12345:4:1
+  LOCAL_PROC_FID0      = 0x7200000000000001:0x29
```

Comment:
1. Why do we need a RC file?
To be simple, just pass them from Command Line.

2. Why do we need separate RC file for m0iscdemorc and m0iscreg?
    Because they will run on different nodes? 
    m0iscdemorc is running from client nodes, and m0iscreg is running from server nodes?

Reply:
This is the legacy from [c0appz](https://github.com/Seagate/cortx-motr-apps) infrastructure. All utilities have configuration files per node. Yes, because they can be run from different nodes which share the same home directory via NFS.

Reply:
Got rid of the RC file and updated the README.